packages/components/src/unit-control/styles/unit-control-styles.ts

```diff
@@ -36,7 +36,7 @@ export const ValueInput = styled( NumberControl )`
 
 const baseUnitLabelStyles = ( { selectSize }: SelectProps ) => {
 	const sizes = {
-		default: css`
+		small: css`
```

Comment:
Changed the naming from `default`/`large` to `default`/`small` to reflect the new sizing scheme.